xds/src/main/java/io/grpc/xds/VirtualHost.java

```diff
@@ -90,6 +89,47 @@ static RouteMatch create(PathMatcher pathMatcher,
         return new AutoValue_VirtualHost_Route_RouteMatch(pathMatcher,
             ImmutableList.copyOf(headerMatchers), fractionMatcher);
       }
+
+      /** Matcher for HTTP request path. */
+      @AutoValue
+      abstract static class PathMatcher {
```

Comment:
May put this `PathMatcher` in `internal.Matchers` as well?

Reply:
I remember we discussed that it is just in router so it stays in router unless things have changed.